Library/Homebrew/utils/notability.rb

```diff
@@ -5,12 +5,14 @@
 module SharedAudits
   module_function
 
+  def github_repo_data(user, repo)
```

Comment:
Wondering if this should be cached so we're not querying the same repo again and again. Thoughts?